src/components/Header.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Title } from 'styledComponents';
+
+const Header = () => {
+  return <Title size="64">Shop the market and get insurance right</Title>;
```

Comment:
Changed this to `size` instead of `fontSize` otherwise it gets added to the dom as `font-size`